week-1/exercises/decimal.js

```diff
@@ -1,9 +1,15 @@
-const num = 58.4567;
+const num = 56.4567;
 
 // You should look up Math functions for this exercise https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math
 
 // Create a variable called wholeNumberPart and assign to it an expression that evaluates to 56 ( the whole number part of num )
 // Create a variable called decimalPart and assign to it an expression that evaluates to 0.4567 ( the decimal part of num )
 // Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
 
+const wholeNumberPart = Math.floor(num);
+const decimalPart = (num - Math.trunc(num)).toFixed(4);
```

Comment:
this is fine, but as you already have `wholeNumberPart` assigned, you could have used that instead, e.g. `const decimalPart = (num - wholeNumberPart)`